roles/rollback/tasks/main.yml

```diff
@@ -15,8 +15,19 @@
     msg: "Cannot switch to release at {{ new_release_path }}. It is from an unfinished deploy. You may manually specify a different release using --extra-vars='release=12345678901234'."
   when: target.stat.exists | default(False)
 
+- name: Get real path of current symlinked release
+  command: "readlink {{ project_current_path }}"
+  args:
+    chdir: "{{ project_root }}"
+  register: current_release_real_path
+
 - name: Link 'current' directory to target release
   file:
     path: "{{ project_root }}/{{ project_current_path }}"
     src: "{{ new_release_path }}"
     state: link
+
+- name: Write unfinished file to old symlinked release
+  file:
+    path: "{{ current_release_real_path.stdout }}/DEPLOY_UNFINISHED"
```

Comment:
`deploy_helper.unfinished_filename` is not available during rollback.
Therefore, I hardcoded it.